libs/blocks/chart/chart.js

```diff
@@ -78,6 +78,40 @@ const barSeriesOptions = (seriesData, colors, unit) => (
   }))
 );
 
+const getContainerSize = (chartSize, chartType) => {
+  const containerSizes = {
+      area: {
+          small: { height: 345, width: '100%' },
+          medium: { height: 310, width: '100%' },
+          large: { height: 512, width: '100%' },
+      },
+      default: {
+          small: { height: 290, width: '100%' },
+          medium: { height: 295, width: '100%' },
+          large: { height: 350, width: '100%' },
+      },
+      donut: {
+          small: { height: 345, width: '100%' },
+          medium: { height: 450, width: '100%' },
+          large: { height: 512, width: '100%' },
+      },
+      oversizedNumber: {
+          small: { minHeight: 290, width: '100%' },
+          medium: { minHeight: 295, width: '100%' },
+          large: { minHeight: 350, width: '100%' },
+      },
+  };
+  let containerSize;
+
+  if (chartType in containerSizes) {
+    containerSize = (typeof containerSizes[chartType][chartSize] === 'object' && !Array.isArray(containerSizes[chartType][chartSize])) ? containerSizes[chartType][chartSize] : {};
+  } else {
+    containerSize = (typeof containerSizes.default[chartSize] === 'object' && !Array.isArray(containerSizes.default[chartSize])) ? containerSizes.default[chartSize] : {};
```

Comment:
Optional chaining is simpler
```
containerSize = containerSizes?.[chartType]?.[chartSize] || {};

Reply:
You may also decide to remove the else by instantiating `containerSize` with the value found in the `else` of your conditional, and then only overwriting when the conditions are met. This will also make it more clear as to what is expected from the return statement on line `112`.

Reply:
Definitely much simpler, thank you.

Reply:
That's a good point, have removed the `else` statement.